services/management/backup/locations_service.go

```diff
@@ -129,6 +129,18 @@ func convertLocation(location *models.BackupLocation) (*backupv1beta1.Location,
 	return loc, nil
 }
 
+// RemoveLocation removes backup location.
+func (s *LocationsService) RemoveLocation(ctx context.Context, req *backupv1beta1.RemoveLocationRequest) (*backupv1beta1.RemoveLocationResponse, error) {
+	mode := models.RemoveRestrict
+	if req.Force {
```

Comment:
JFYI it's unlikely to have something in the API which may confuse users.
We can keep it for now here, because it's private API.

Reply:
What do you mean?
There is another ticket https://jira.percona.com/browse/PMM-7475
There will be backup inventory (performed backup on some location). Force delete should allow remove this location even though there was already made some backup.

But I agree, that it probably should not be cascade.

Reply:
I mean that API says that `force: true` deletes backup location with all dependencies, but actually this flag is ignored until PMM-7475 is implemented.

Reply:
i wanted to use `unwanted` word instead of `unlikely`